projects/go-tester/src/app/components/test-page-1/test-page-1.component.html

```diff
@@ -1,3 +1,8 @@
+<go-header-bar title="This is the title" [showBackArrow]="true">
+  <go-button go-header-bar-action>Hello</go-button>
```

Comment:
Does each one of these `go-header-bar-action` items get rendered into the `ng-content` of the component? Also, it would probably be easier to format these using the button group classes, https://tangoe.design/ui-kit/button#button-group

I know this is just the test project, but still.